modules/users/server/controllers/users/users.authentication.server.controller.js

```diff
@@ -116,7 +116,7 @@ exports.oauthCallback = function (strategy) {
           return res.redirect('/authentication/signin');
         }
 
-        return res.redirect(redirectURL || sessionRedirectURL || '/');
```

Comment:
We're always open for improvements. Go ahead and open a PR and let's see what's on the table.
We already had a big refactoring to the authentication and authorization code one time.